packaging/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ install(TARGETS Halide Halide_Generator Halide_LanguageOptions
 
 if (TARGET Halide_Adams2019)
     install(TARGETS Halide_Adams2019 Halide_Li2018 Halide_Mullapudi2016
```

Comment:
This doesn't seem quite right, but might be okay to get people unstuck... it makes me nervous, though.

The package decomposition I want in the end is as follows:

1. `HalideHelpers` (interfaces) includes all the platform-independent CMake helpers, header-only/source-injection libraries, etc.
2. `HalideBuildTools` includes all the libraries and executables intended to run on the host system. This is where the (to-be-created) `gengen` tool and the autoschedulers should go in the end.
3. `Halide` includes the libraries meant to be linked into the target applications.

Reply:
It's good to know that there is a plan and design which seems to make more sense. I admit this patch is a kind of hack that may not follow the existing design intension. My short term goal is to make it possible to compile the application like #7286 by CMake. If there is any other approach that could be less unacceptable, I'm happy to try and test.